x-pack/plugins/enterprise_search/public/applications/shared/schema/add_field_modal/index.tsx

```diff
@@ -39,14 +39,14 @@ import {
 } from './constants';
 import { formatFieldName } from './utils';
 
-interface ISchemaAddFieldModalProps {
-  disableForm?: boolean;
-  addFieldFormErrors?: string[] | null;
+interface Props {
```

Comment:
Just FYI, it looks like we use a similar convention [here](https://github.com/constancecchen/kibana/blob/shared-schema-4/x-pack/plugins/enterprise_search/public/applications/shared/schema/schema_errors_accordion.tsx#L37).

Maybe something to clean up at some point in the schema work.

Reply:
With all my currently open PRs, I should have cleaned up all the various shared schema `Props` interfaces! https://github.com/elastic/kibana/pull/99069/files#diff-1e84fd516379972471dccc28de549ae25b33e7841b593db249b249af8679a239R36 😁